lightwood/encoders/time_series/rnn.py

```diff
@@ -28,18 +28,24 @@ def __init__(self, encoded_vector_size=4, train_iters=100, stop_on_error=0.01, l
         self._eos = 0.0  # end of input sequence -- padding value for batches
         self._n_dims = ts_n_dims
         self._normalizer = None
+        self._target_ar_normalizer = None
         self._criterion = nn.MSELoss()
 
-    def setup_nn(self):
+    def setup_nn(self, additional_targets=None):
```

Comment:
Why do we need to pass the `additiona_targets` here ? As in, since the timeseries encoder can handle arrays of arbitrary sizes would it not make more sense not to have any "special" logic around previous target values ?

Think, for example, of a case where we not only want to encoder the order column + the previous targets, but rather jointly encode 3x or 4x different columns with the same RNN encoder, this approach wouldn't scale to that.

Reply:
An alternative would be to pass the `Transformer.feature_len_map` instead